training/replication-rebalancing-fault-tolerance.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Replication, Rebalancing & Fault Tolerance
+summary: Learn how CockroachDB replicated and rebalances your data to survive failures.
+toc: false
+sidebar_data: sidebar-data-training.json
+---
+
+<div id="toc"></div>
+
+## Presentation
+
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vQXJYTZky37sze5ZKd_xfSuK_bqMWDbMCNVIWM1h4s6rtoQqpNzM2drT4ZQGbBsUJefwwaY3cmEQe6A/embed?start=false&loop=false" frameborder="0" width="756" height="454" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+#### URL for comments
+
+[Replication, Rebalancing &amp; Rebalancing](https://docs.google.com/presentation/d/1rOSpwKD48T4OsFRFwnrRo-7x6dLK9Gr2pg9oBRTaChg/edit#slide=id.g18624943e3_0_0)
+
+## Lab
+
+In this lab, we'll show you that CockroachDB is a fault-tolerant database by taking down one of the nodes and showing that the cluster keeps operating.
+
+### Before You Start
+
+To complete this lab, we assume you have [a 3-node cluster and a load generator running](3-node-local-insecure-cluster-load-gen.html).
```

Comment:
I'm just trying to make it as quick and easy for users to get caught up. My thought was that people might feel like they _need_ to skim over the presentation if it's on the page, rather than just slogging through the code samples really quickly.